src/commandline_content.ts

```diff
@@ -47,19 +50,18 @@ async function init() {
 
 // Load the iframe immediately if we can (happens if tridactyl is reloaded or on ImageDocument)
 // Else load lazily to avoid upsetting page JS that hates foreign iframes.
-try {
-    init()
-} catch (e) {
-    // Surrender event loop with setTimeout() to page JS in case it's still doing stuff.
-    document.addEventListener("DOMContentLoaded", () => setTimeout(init, 0))
-}
 
-export function show() {
+export async function show() {
     try {
+        await init()
+        await hide()
```

Comment:
This is the real travesty in this PR, but it makes it kinda work.

Reply:
Yeah, this is odd.